media/transcoder/transcoder_test.go

```diff
@@ -0,0 +1,352 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+)
+
+const (
+	location              = "us-central1"
+	templateID            = "my-go-test-template"
+	deleteTemplateReponse = "Deleted job template"
+	deleteJobReponse      = "Deleted job"
+	jobSucceededState     = "SUCCEEDED"
+	testVideoFileName     = "ChromeCast.mp4"
+	testVideoFileLocation = "../testdata/"
+	bucketName            = "golang-samples-transcoder-test"
```

Comment:
Will you update this bucket name to include the project name? That will require making them dynamic, rather than top-level consts. That way, we avoid issues with the CI running multiple versions of Go in parallel and causing race conditions.

(This is the cause of the latest Kokoro failure.)